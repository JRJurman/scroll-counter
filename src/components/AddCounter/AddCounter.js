import { registerHtml } from 'tram-one'
import './AddCounter.scss'

const html = registerHtml()

export default (props) => {
  return html`
    <button class="AddCounter" ${props} >
      +
    </button>
  `
}
