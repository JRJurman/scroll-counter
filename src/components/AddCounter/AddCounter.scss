.AddCounter {
  font-size: inherit;
  width: 100%;
}
