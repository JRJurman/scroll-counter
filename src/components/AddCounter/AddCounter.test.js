import AddCounter from './AddCounter'

describe('AddCounter', () => {
  describe('with default props', () => {
    const wrapper = AddCounter()

    it('should match snapshot', () => {
      expect(wrapper.outerHTML).toMatchSnapshot()
    })
  })
})
