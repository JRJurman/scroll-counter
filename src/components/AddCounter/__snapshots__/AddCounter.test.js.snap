// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddCounter with default props should match snapshot 1`] = `
"<button undefined=\\"undefined\\" class=\\"AddCounter\\">
      +
    </button>"
`;
